src/result/__init__.py

```diff
@@ -8,5 +18,7 @@
     "UnwrapError",
     "as_async_result",
     "as_result",
+    "is_ok",
+    "is_err",
```

Comment:
These technically will conflict with the `Err.is_ok` and `Ok.is_ok` methods.
Might cause some confusion?
But as long as someone is using a type checker with this library, they'll realize those methods aren't type guards and will use the proper `is_ok`/`is_err` type guard functions...hopefully I guess.